packages/components/src/confirm-dialog/test/index.js

```diff
@@ -34,6 +35,33 @@ describe( 'Confirm', () => {
 					expect( el ).toBeInTheDocument();
 				} );
 			} );
+			it( 'should render correctly with custom button lables', () => {
```

Comment:
Small typo here: `lables` -> `labels` :)

Reply:
Thank you! (and thanks for the fix!)